content/blog/2021-06-15-june-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,239 @@
+﻿---
+title: June '21 Heartbeat
+date: 2021-06-16
+description: |
+  Monthly updates are here! Making sense of the MLOps Landscape, MLOps 
+  Community love, our MLOps philosophy, DVC Studio , R for DVC, new learning
```

Comment:
Does `DVC Studio ,` need to be `DVC Studio,`?

Reply:
Yep!